src/main/java/com/minecolonies/coremod/colony/permissions/ForgePermissionNodes.java

```diff
@@ -0,0 +1,76 @@
+    BACKUP ("com.minecolonies.coremod.Backup", DefaultPermissionLevel.OP, "Can back up colony data."),
+    LIST_CITIZENS ("com.minecolonies.coremod.ListCitizens", DefaultPermissionLevel.ALL, "Can ListCitizens...."),
+    KILL_CITIZEN ("com.minecolonies.coremod.KillCitizen", DefaultPermissionLevel.ALL, "Can KillCitizen...."),
+    RESPAWN_CITIZEN ("com.minecolonies.coremod.RespawnCitizen", DefaultPermissionLevel.ALL, "Can RespawnCitizen...."),
+    CITIZEN_INFO ("com.minecolonies.coremod.CitizenInfo", DefaultPermissionLevel.ALL, "Can view detailed citizen info."),
+    CHANGE_COLONY_OWNER ("com.minecolonies.coremod.ChangeColonyOwner", DefaultPermissionLevel.OP, "Can change owner of a colony."),
+    SHOW_COLONY_INFO ("com.minecolonies.coremod.ShowColonyInfo", DefaultPermissionLevel.OP, "Can ShowColonyInfo...."),
+    DELETE_COLONY ("com.minecolonies.coremod.DeleteColony", DefaultPermissionLevel.OP, "Can DeleteColony...."),
+    DISABLE_BARBARIAN_SPAWNS ("com.minecolonies.coremod.DisableBarbarianSpawns", DefaultPermissionLevel.OP, "Can DisableBarbarianSpawns...."),
+    ADD_OFFICER ("com.minecolonies.coremod.AddOfficer", DefaultPermissionLevel.OP, "Can AddOfficer...."),
+    REFRESH_COLONY ("com.minecolonies.coremod.RefreshColony", DefaultPermissionLevel.OP, "Can RefreshColony...."),
+    COLONY_TELEPORT ("com.minecolonies.coremod.ColonyTeleport", DefaultPermissionLevel.OP, "Can ColonyTeleport...."),
+    MAKE_NOT_AUTO_DELETABLE ("com.minecolonies.coremod.MakeNotAutoDeletable", DefaultPermissionLevel.OP, "Can MakeNotAutoDeletable...."),
+    DO_RAID_NOW ("com.minecolonies.coremod.DoRaidNow", DefaultPermissionLevel.OP, "Can DoRaidNow...."),
+    DO_RAID_TONIGHT ("com.minecolonies.coremod.DoRaidTonight", DefaultPermissionLevel.OP, "Can DoRaidTonight...."),
+    REQUEST_SYSTEM_RESET ("com.minecolonies.coremod.RSReset", DefaultPermissionLevel.OP, "Can RSReset...."),
+    BARBARIAN_KILL ("com.minecolonies.coremod.BarbarianKill", DefaultPermissionLevel.OP, "Can BarbarianKill...."),
+    ANIMAL_KILL ("com.minecolonies.coremod.AnimalKill", DefaultPermissionLevel.OP, "Can AnimalKill...."),
+    MOB_KILL ("com.minecolonies.coremod.MobKill", DefaultPermissionLevel.OP, "Can MobKill...."),
+    CHICKEN_KILL ("com.minecolonies.coremod.ChickenKill", DefaultPermissionLevel.OP, "Can ChickenKill...."),
+    COW_KILL ("com.minecolonies.coremod.CowKill", DefaultPermissionLevel.OP, "Can CowKill...."),
+    PIG_KILL ("com.minecolonies.coremod.PigKill", DefaultPermissionLevel.OP, "Can PigKill...."),
+    SHEEP_KILL ("com.minecolonies.coremod.SheepKill", DefaultPermissionLevel.OP, "Can SheepKill...."),
+    SCAN ("com.minecolonies.coremod.Scan", DefaultPermissionLevel.OP, "Can scan structures."),
+    ;
+
+    @NotNull final private String nodeName;
+    @NotNull final private DefaultPermissionLevel defaultPermissionLevel;
+    @NotNull final private String description;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)